docs/content/guides/developer/app-examples/reviews-rating.mdx

```diff
@@ -0,0 +1,212 @@
+---
+title: Reviews Rating
+---
+
+The following documentation goes through an example implementation of a review rating platform for the food service industry on Sui.
+Unlike traditional review rating platforms, that often do not disclose the algorithm used to rate reviews, the on-chain platform uses an algorithm that is published on-chain for everyone to see and verify.
+Furthermore, all submitted reviews are scored and ordered on-chain which is feasible due to the low gas cost of computation on Sui.
+
+## Personas
+
+There are four actors in the typical workflow of the reviews rating example.  
+
+```mermaid
+sequenceDiagram
+  Service ->> Dashboard: Add service to dashboard.
+  Service ->> User: Send proof of experience.
+  User ->> Service: Send review
+  Service ->> User: Send reward
+  Service ->> Moderator: Add as a service moderator
+  Moderator ->> Service: Remove abused review
+```
+
+### Service owners
+
+Service owners are entities like restaurants that list their services on the platform. Service owners allocate a specific amount of SUI as a reward pool. Assets from the pool are used to provide rewards for high-rated reviews. A Proof of Experience (PoE) NFT confirms the reviewer used the service, which they burn to provide a verified review. Service owners can provide their customers with unique identifiers, perhaps using QR codes, to identify individual reviewers.
+
+### Reviewers
+
+Reviewers are consumers of services that use the review system. Reviewers provide feedback in the form of comments that detail specific aspects of the service as well as a star rating to inform others. The reviews are rated, with the most effective reviews getting the highest rating. Service owners award the 10 highest rated reviews for their service. How often the rewards are distributed is up to the service owner; it can be on a regular or rolling basis.
```

Comment:
It's not clear to me what the distinction is between "on a regular basis" or "on a rolling basis"? They sound like they could mean the same thing, could you add more detail here to distinguish the two modes (or is the intention that the rewards are either distributed as a one-off vs a rolling basis?)

Reply:
I've made it clear that it is entirely up to service owner that it could daily or month